submissions/cumiOyemike/ADS/index.html

```diff
@@ -7,7 +7,11 @@
     <title>Document</title>
 </head>
 <body>
+<<<<<<< HEAD:submissions/seyifunmiOjo/Javascript/arrays/index.html
+    <h1>Your first Javascript Array</h1>
+=======
     <h1>Algorithms & Data Structures 1</h1>
     <script src=index.js></script>
+>>>>>>> master:submissions/cumiOyemike/ADS/index.html
```

Comment:
Remove this on line 15

Reply:
@adaugochi this is another person's file and not mine ma?

Reply:
oh..  I see

Reply:
Kindly remove the merge conflict that I specify on line 10, 12 and 15. It was because of the conflict you resolved that added those things to her code. kindly remove it and push again